docs/manuals/run-deploy/deploy/linux/ubuntu.md

```diff
@@ -64,52 +64,54 @@ $python app.py
 <-- Truncate -->
 ```
 
-The application is running locally, you can access it with the browser on the URL [http://127.0.0.1:5000/](http://127.0.0.1:5000/).
+The application is running locally, you can access it with the browser on the URL
+[http://127.0.0.1:5000/](http://127.0.0.1:5000/).
 
 !!! note
     The message:
     ```
     WARNING: This is a development server. Do not use it in a production deployment.
     Use a production WSGI server instead.
     ```
-    Is provided by Flask because the way of exposing an application on the Internet and
+    Is issued by Flask because the way of exposing an application on the Internet and
```

Comment:
Should we add that this warning turns into an error when flask > 2.2.x ie is then blocking?

Reply:
Will do. Thx